docs/topics/auth/default.txt

```diff
@@ -150,7 +150,7 @@ Authenticating users
 Permissions and Authorization
 =============================
 
-Django comes with a simple permissions system. It provides a way to assign
+Django comes with an inbuilt permissions system. It provides a way to assign
```

Comment:
"builtin" is the phrase used in the docs, "inbuilt" isn't used anywhere at the moment.